libcloud/common/exceptions.py

```diff
@@ -47,7 +51,14 @@ class RateLimitReachedError(BaseHTTPError):
     message = '%s Rate limit exceeded' % (code)
 
     def __init__(self, *args, **kwargs):
-        self.retry_after = int(kwargs.pop('retry_after', 0))
+        headers = kwargs.pop('headers', None)
+        super(RateLimitReachedError, self).__init__(self.code,
+                                                    self.message,
+                                                    headers)
```

Comment:
This doesn't look right, I don't think `self.code` and `self.message` are bound yet?  That's the point of calling the superclass constructor...

Reply:
And if that's really a bug, the test coverage needs to catch it.

Reply:
I see now, they are class variables, so that's okay.  Nevermind.